docs/user/build/install/jdk.rst

```diff
@@ -48,23 +41,37 @@ Java 5    GeoTools 2.5.x   GeoTools 8.x     compiler=1.5
 Java 1.4  GeoTools 2.x     GeoTools 2.4.x   compiler=1.4 
 ========= ================ ================ =================
 
-Using Java 9 with GeoTools
-''''''''''''''''''''''''''
+Using Java 11 with GeoTools
+'''''''''''''''''''''''''''
+
+Java introduced a number of changes to the JVM, most notably the module system (Project Jigsaw). Refer to `The State of the Module System <http://openjdk.java.net/projects/jigsaw/spec/sotms/>`_ for more details.
+
+========= ================ ================ =================
+Java      Initial          Final            Compiler Setting 
+========= ================ ================ =================
+Java 11   GeoTools 21.x    And Above        compiler=1.8 
+========= ================ ================ =================
+
+If your project depends on GeoTools 21.x, and you want to use it with Java 11:
 
-Java 9 introduces a number of changes to the JVM, most notably the module system (Project Jigsaw). Refer to `The State of the Module System <http://openjdk.java.net/projects/jigsaw/spec/sotms/>`_ for more details.
-While GeoTools has not yet been updated to use the module system, GeoTools 19.x can run on Java 9.
+* Use the GeoTools jars on the module path, they are configured with automatic module names
 
-If your project depends on GeoTools 19.x, and you want to use it with Java 9, you will need to add the following flags to your JVM runtime arguments::
+* Use the GeoTools jars on the classpath, adding the following flags to your JVM runtime arguments::
 
     --add-modules=java.xml.bind --add-modules=java.activation
```

Comment:
This shouldn't be necessary anymore; we have these modules as proper dependencies now

Reply:
Yeah, but they aren't part of the JVM anymore, there standalone dependencies, so either way we shouldn't need this runtime argument. GeoTools/GeoServer works fine without it (and it runs on the classpath)